### PR TITLE
[Symfony Bridge][Extension] added EventDispatcherExtension.

### PR DESCRIPTION
**Pull request by aitboudad**

### CODE REVIEW

File: src/Payum/Core/Bridge/Symfony/Event/ReplyEvent.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+     */
+    public function getReply()
+    {
+        return $this->reply;
+    }
+
+    /**
+     * @param ReplyInterface $reply
+     */
+    public function setReply(ReplyInterface $reply)
```
Comment (staabm) [inline src/Payum/Core/Bridge/Symfony/Event/ReplyEvent.php]: Why a setter?

**Comment by mkurzeja** [comment]
Any plans to merge this? Would be great to have it

**Comment by makasim** [comment]
The extension api was changed so the PR has to be updated. The idea is good though.

**Comment by mkurzeja** [comment]
@aitboudad interested in updating this PR? If not I could try to find some time and work on it.

**Comment by aitboudad** [comment]
@mkurzeja not sure, if you have some time why not :)

**Pull request merged**